src/app/App.js

```diff
@@ -102,6 +115,15 @@ function App() {
               <UserPage onUserChange={handleUserChange} user={user} />
             )}
           />
+          <Route
+            path="/info"
```

Comment:
path is `/Info` but renders the `medical page`, maybe change the path to reflect semantically what is shown